README.md

```diff
@@ -4,14 +4,25 @@ A list of dead JIT compiler projects for CPython.
 > Those who fail to learn from history are doomed to repeat it.
 
   Sir Winston Churchill
-  
+
+### Psyco
+
+An importable extension module that did basic JIT of the eval loop calling CPython
+APIs similar to what MyPyC and Cython might do without any type annotations. 32-bit
```

Comment:
Psyco was more sophisticated than that. http://psyco.sourceforge.net/theory_psyco.pdf

Reply:
thanks, simplified the wording.